### DISCUSSION TITLE
Uncaught Error: Interface "SessionHandlerInterface" not found

### DISCUSSION THREAD

**Discussion by sjempotje** [bs-community/blessing-skin-server#530] [Q&A] [answered]
Hello,

Can somebody help me with this problem:

2023/01/21 21:29:28 [error] 19#19: *1 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Uncaught Error: Interface "SessionHandlerInterface" not found in /home/container/webroot/vendor/laravel/framework/src/Illuminate/Session/FileSessionHandler.php:10

Stack trace:

#0 /home/container/webroot/vendor/composer/ClassLoader.php(571): include()

#1 /home/container/webroot/vendor/composer/ClassLoader.php(428): Composer\Autoload\includeFile()

#2 /home/container/webroot/vendor/laravel/framework/src/Illuminate/Session/SessionManager.php(73): Composer\Autoload\ClassLoader->loadClass()

#3 /home/container/webroot/vendor/laravel/framework/src/Illuminate/Session/SessionManager.php(61): Illuminate\Session\SessionManager->createNativeDriver()

#4 /home/container/webroot/vendor/laravel/framework/src/Illuminate/Support/Manager.php(105): Illuminate\Session\SessionManager->createFileDriver()

#5 /home/container/webroot/vendor/laravel/framework/src/Illuminate/Support/Manager.php(80): Illuminate\Support\Manager->createDriver()

#6 /home/container/webroot/vendor/laravel/framework/sr...PHP message: PHP Fatal error:  Uncaught Error: Interface "SessionHandlerInterface" not found in /home/container/webroot/vendor/laravel/framework/src/Illuminate/Session/FileSessionHandler.php:10

**Comment by CharCaius** [comment, marked as answer]
You shouldn't ask questions in the Discussion. It is recommended to open an Issue or join a Telegram group.

**Reply by sjempotje** [reply]
Thanks! Made one but I was able to fix it myself.

**Comment by 0AvA0** [comment]
Hello, I also encountered this problem, please how should I solve it.
![image](https://user-images.githubusercontent.com/89952068/217409537-066b72d2-337e-494f-8405-1b936206b566.png)
